workflows/DATA_TO_MODEL.nf

```diff
@@ -3,51 +3,73 @@
 nextflow.enable.dsl = 2
 // Empty string default to avoid warning
 params.submissions = ""
-// Synapse ID for Submission View
-params.view_id = "syn52576179"
-// Scoring Script
-params.scoring_script = "data_to_model_score.py"
-// Validation Script
-params.validation_script = "validate.py"
-// Testing Data
-params.testing_data = "syn51390589"
+// The expected file type for the submissions
+params.file_type = "csv"
+// The command used to execute the Challenge scoring script in the base directory of the challenge_container: e.g. `python3 path/to/score.py`
+params.execute_scoring = "python3 /home/user/score.py"
+// The command used to execute the Challenge validation script in the base directory of the challenge_container: e.g. `python3 path/to/validate.py`
+params.execute_validation = "python3 /home/user/validate.py"
 // E-mail template (case-sensitive. "no" to send e-mail without score update, "yes" to send an e-mail with)
 params.email_with_score = "yes"
-// Ensuring correct input parameter values
-assert params.email_with_score in ["yes", "no"], "Invalid value for ``email_with_score``. Can either be ''yes'' or ''no''."
 // toggle email notification
 params.send_email = true
 // set email script
 params.email_script = "send_email.py"
 
+// Ensuring correct input parameter values
+params.file_type_lower = params.file_type?.toLowerCase()
```

Comment:
I changed the name to `file_type_lower` because I caught a warning saying that referencing a parameter twice leads to Nextflow just picking the first reference, which we don't want.

This[ workflow run](https://tower-dev.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/example-dev-project/watch/x15MNuofZnO0m) confirms that enforcing the lower-case works for sure.